syft/formats/common/spdxhelpers/to_format_model_test.go

```diff
@@ -13,9 +15,100 @@ import (
 	"github.com/anchore/syft/syft/file"
 	"github.com/anchore/syft/syft/pkg"
 	"github.com/anchore/syft/syft/sbom"
+	"github.com/anchore/syft/syft/source"
 )
 
-// TODO: Add ToFormatModel tests
+func Test_toFormatModel(t *testing.T) {
+	tests := []struct {
+		name     string
+		in       sbom.SBOM
+		expected *spdx.Document
+	}{
+		{
+			name: "includes root element",
+			in: sbom.SBOM{
+				Source: source.Description{
+					Metadata: source.StereoscopeImageSourceMetadata{
+						UserInput: "alpine:latest",
+						ID:        "sha256:d34db33f",
+					},
+				},
+				Artifacts: sbom.Artifacts{
+					Packages: pkg.NewCollection(pkg.Package{
+						Name:    "pkg-1",
+						Version: "version-1",
+					}),
+				},
+			},
+			expected: &spdx.Document{
+				SPDXIdentifier: "DOCUMENT",
+				SPDXVersion:    spdx.Version,
+				DataLicense:    spdx.DataLicense,
+				DocumentName:   "alpine:latest",
+
+				Packages: []*spdx.Package{
+					{
+						PackageSPDXIdentifier: "Package--pkg-1-pkg-1",
```

Comment:
nit: should we be able to render `Package--...` if information is missing? (I feel like it should be `Package-...`)